scout/adapter/mongo/query.py

```diff
@@ -353,6 +353,8 @@ def affected_inds_query(self, mongo_query, case_id):
             "$elemMatch": {"$or": []}  # Any of the affected individuals should exibit the variant
         }
         case_obj = self.case(case_id=case_id)
+        if len(case_obj.get("individuals"), []) == 1:  # No point in adding this filter
```

Comment:
💯 But probably hide the checkbox then as well, right!

Reply:
I thought about it, but I haven't removed it from the template for 3 reasons:

1. When they are in a case with only one individual, people might wonder if they are seeing all variants or not (of course they are seeing them all, but you know..)
2. To keep the form consistent in all cases, which is also part of point 1
3. Because then you have to pass the case object to this macro, and this macro is invoked from different forms: the SNVs form, the SVs form, the cancer vars form, ??. So it's a lot of additional code that I wasn't really sure was worth to add. Also because I'm lazy..

So unless you really want to hide I would leave it like that.. ?

Reply:
<img width="303" alt="image" src="https://user-images.githubusercontent.com/28093618/156596850-cec7110d-bbda-4e27-be39-8e3baef69ddc.png">